conda-store-server/conda_store_server/server/auth.py

```diff
@@ -188,14 +188,17 @@ def convert_roles_to_permissions(self, roles):
             permissions = permissions | self.role_mappings[role]
         return permissions
 
-    def get_entity_binding_permissions(self, entity):
+    def convert_namespace_to_entity_arn(self, namespace):
+        return f"{namespace}/*"
```

Comment:
I didn't want to change the auth part too much, so I just convert to a compatible format here. Namespace role mappings use concrete roles (like, "viewer") rather than this ARN (regex) format. You can think of a namespace as a username.

Reply:
Looks great to me.